spec/functions/pw_hash_spec.rb

```diff
@@ -52,6 +52,7 @@
 
   context 'when the third argument contains invalid characters' do
     it { is_expected.to run.with_params('password', 'sha-512', 'one%').and_raise_error(ArgumentError, %r{characters in salt must be in the set}) }
+    it { is_expected.to run.with_params('password', 'bcrypt', '1234').and_raise_error(ArgumentError, %r{characters in salt must match}) }
```

Comment:
While this look's good overall I would prefer it if you were to expand the added test's to include the other new algorithms.

Reply:
Sure, I could do that, but I'm not convinced it is worthwhile since the test is just for bad salt values and the other three tests would be ~identical.  I would love to test the hash methods more properly, but as mentioned in commit f3800aa, the test system's Ruby doesn't support them :/

The salient error from the spec test was:

```
expected pw_hash("password", "bcrypt", "05$abcdefghijklmnopqrstuv") to have returned /^\$2b\$05\$abcdefghijklmnopqrstu/ instead of raising Errno::EINVAL(Invalid argument - crypt)
```